src/workerd/io/worker-interface.capnp

```
@@ -482,6 +488,14 @@ struct JsValue {
   }
 }
 
+interface AbortSignal $Cxx.allowCancellation {
```

It occurs to me that we may want to think carefully about what happens if the AbortSignal is dropped without calling `triggerAbort()`.

The intuitive answer, from a JavaScript perspective, is that nothing should happen. The signal never aborts.

But the standard answer from a Cap'n Proto perspective is that we should assume an abort. This is because one reason the capability might be dropped is due to a network disconnect. In this case, the client has lost the ability to communicate aborts to us, but very likely, if they've lost the connection, they'd prefer to cancel the operation. "Fail closed" rather than "fail open".

To this end we may actually want this RPC interface to have two methods: `abort()` and `release()`. Calling `release()` means "the AbortController was GC'd on the client side, indicating the client never intends to abort it, so please do NOT assume an abort when this capability is dropped, which it will be shortly". Only if we didn't receive a `release()`, we would assume an abort when dropped.